src/main/java/mat/server/PreventCachingFilter.java

```diff
@@ -62,18 +62,19 @@ else if(requestURI.indexOf("/Mat.html") >= 0 || requestURI.indexOf("/Bonnie.html
 			httpResponse.setDateHeader("Expires", 0);
 			httpResponse.setHeader("Pragma", "no-cache");
 			httpResponse.setHeader("Cache-control", "no-cache, no-store, must-revalidate");
-			if(LoggedInUserUtil.getLoggedInUser() == null) {
-				logger.info("Redirecting request for " + httpRequest.getRequestURI() + " to Login in session " + httpRequest.getSession().getId());
-				httpResponse.setStatus(302);
-				String url = "Login.html";
-				if(httpRequest.getQueryString() != null) {
-					url = url + "?" + httpRequest.getQueryString();
-				}
-				httpResponse.setHeader("Location", url);
-			}
-			else {
+			//TODO Check if we need to do ALL of this no-cache and re-direct stuff.
+//			if(LoggedInUserUtil.getLoggedInUser() == null) {

```

Comment:
TODO left open

Reply:
Thanks. Updated TODO with jira ticket as it is not part of this task.